whatwg.org/chat

```diff
@@ -27,38 +27,44 @@
  <a href="https://participate.whatwg.org/">Participate</a>
 </nav>
 
-<h3 id="irc">IRC<a class="self-link" href="#irc"></a></h3>
+<h3 id="chat">Chat<a class="self-link" href="#irc"></a></h3>
 
-<p>The Freenode IRC network has a channel called <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> where some of the more active WHATWG community members hang out. Feel free to join us!</p>
+<p>We have both a <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> channel on the Freenode IRC network and a <a href="https://app.element.io/#/room/#whatwg:matrix.org">whatwg</a> room in the <a href="https://en.wikipedia.org/wiki/Matrix_(protocol)">Matrix</a> network. Join us!</p>
```

Comment:
I'd also like to turn this sentence around, something like:
```
<p>We have a  a <a href="https://app.element.io/#/room/#whatwg:matrix.org">whatwg</a> room in the <a href="https://en.wikipedia.org/wiki/Matrix_(protocol)">Matrix</a> network, as well as an older <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> channel on the Freenode IRC network. Join us!</p>

Reply:
> I'd also like to turn this sentence around, something like:
> 
> ```
> <p>We have a  a <a href="https://app.element.io/#/room/#whatwg:matrix.org">whatwg</a> room in the <a href="https://en.wikipedia.org/wiki/Matrix_(protocol)">Matrix</a> network, as well as an older <a href="ircs://irc.freenode.net/whatwg">#whatwg</a> channel on the Freenode IRC network. Join us!</p>
> ```

OK — made it so